omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -176,9 +177,9 @@ then
     found_manage=true
 fi
 
-if [[ -e /opt/opscode-reporting/bin/opscode-reporting-ctl ]];
+#Use socket to make sure we're on the primary in an HA system
+if [[ -e /opt/opscode-reporting/bin/opscode-reporting-ctl && -S "/tmp/.s.PGSQL.5432" ]];
```

Comment:
Unfortunately this isn't a reliable indicator if reporting (or reporting via chef server) is  configured with external postgres.